From 2d0caa38e040b081903e50faa56bae52599b3949 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Wed, 2 Jul 2014 19:01:22 +0200
Subject: [PATCH 20/20] DOC: provide an example of how to use ssl_c_sha1

As suggested by Aydan Yumerefendi, a little bit of examples never hurts.
---
 doc/configuration.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index fc16843..aa55eb4 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -10793,6 +10793,10 @@ ssl_c_sha1 : binary
   Returns the SHA-1 fingerprint of the certificate presented by the client when
   the incoming connection was made over an SSL/TLS transport layer. This can be
   used to stick a client to a server, or to pass this information to a server.
+  Note that the output is binary, so if you want to pass that signature to the
+  server, you need to encode it in hex or base64, such as in the example below:
+
+     http-request set-header X-SSL-Client-SHA1 %[ssl_c_sha1,hex]
 
 ssl_c_sig_alg : string
   Returns the name of the algorithm used to sign the certificate presented by
-- 
1.8.5.5

